profiler/profiler.go

```diff
@@ -27,10 +27,6 @@ func (p *profiler) AddStats(key []string, value float64) {
 	p.Lock()
 	defer p.Unlock()
 	k := strings.Join(key, "_")
-
-	if _, ok := p.stats[k]; !ok {
-		p.stats[k] = 0
-	}
```

Comment:
Unrelated change.
No need to initialize a new key's value to zero.
If the key is not in the map, then golang will do exactly what this removed code did.